src/tools/kata-ctl/Makefile

```diff
@@ -20,7 +20,7 @@ COMMIT := $(if $(shell git status --porcelain --untracked-files=no 2>/dev/null |
 # Exported to allow cargo to see it
 export KATA_CTL_VERSION := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
-GENERATED_CODE = src/version.rs
+GENERATED_CODE = src/ops/version.rs
```

Comment:
@dborquez is this a rust standard directory for generated code?

Reply:
@cmaf thank you for your inputs, I only chose to group the operations in the same directory, but if you think it is better to get `version.rs` it back to `src/` I will move it back.

Reply:
@dborquez it is fine the way it is, thank you!